quarkus-debezium-parent/quarkus-debezium-postgres-parent/deployment/src/main/java/io/quarkus/debezium/postgres/deployment/QuarkusDatasource.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.quarkus.debezium.postgres.deployment;
+
+import java.net.URI;
+import java.util.Map;
+
+public class QuarkusDatasource {
+    public static Map<String, String> generateDebeziumConfiguration(Map<String, String> properties) {
+        URI uri = URI.create(properties.get("quarkus.datasource.jdbc.url").substring(5));
+
+        return Map.of(
+                "quarkus.debezium.configuration.database.hostname", uri.getHost(),
```

Comment:
I'm wondering if we should just use `quarkus.debezium.*` instead of `quarkus.debezium.configuration.*`.